src/vs/workbench/browser/parts/activitybar/activitybarPart.ts

```diff
@@ -61,28 +61,28 @@ export class ActivitybarPart extends Part implements IActivityService {
 	private registerListeners(): void {
 
 		// Activate viewlet action on opening of a viewlet
-		this.toUnbind.push(this.eventService.addListener(EventType.VIEWLET_OPENING, (e: ViewletEvent) => this.onViewletOpening(e)));
+		this.toUnbind.push(this.eventService.addListener(EventType.VIEWLET_OPENING, (e: CompositeEvent) => this.onViewletOpening(e)));
```

Comment:
@isidorn I think we can generalize this event to be COMPOSITE_OPENING and onCompositeOpening() because the listener seems to use the ID of the composite to do something and it also seems to be the only client through out the workbench.